src/main/kotlin/mate/academy/LetterRemover.kt

```diff
@@ -1,5 +1,8 @@
 package mate.academy
 
-class LetterRemover {
-    // implement ...
+class LetterRemover private constructor() {
+    companion object {
+        @JvmStatic
+        fun removeLetter(input: String) = input.lowercase().map { if (it == 'r') '!' else it }.toCharArray()
```

Comment:
Please check functionality of kotlin's replace() function
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/replace.html